scripts/oec_data_source_script.rb

```diff
@@ -0,0 +1,154 @@
+require 'selenium-webdriver'
+
+begin
+
+  puts "#{Time.now} - Beginning OEC data source update script"
+  args = ARGV.to_s
+  campus_data_sources = %w(Campus_Courses Campus_Supervisors Campus_Instructors Campus_Students Campus_Course_Supervisors Campus_Course_Instructors Campus_Course_Students)
+  extension_data_sources = Dir[File.join(ENV['HOME'], 'selenium-files/*.csv')]
+
+  @base_url = args.include?('qa') ? 'https://course-evaluations-qa.berkeley.edu' : 'https://course-evaluations.berkeley.edu'
+
+  # Set timeouts and launch browser
+  @brief_wait = Selenium::WebDriver::Wait.new(timeout: 30)
+  @medium_wait = Selenium::WebDriver::Wait.new(timeout: 120)
+  @long_wait = Selenium::WebDriver::Wait.new(timeout: 900)
+  @driver = Selenium::WebDriver.for :firefox
+  @driver.manage.window.maximize
+
+  def click_element_id(driver, id)
+    @brief_wait.until { driver.find_element(id: id) }
+    driver.find_element(id: id).click
+  end
+
+  def log_in
+    puts "#{Time.now} - Logging in to Blue"
```

Comment:
@pfarestveit perhaps include @base_url in `puts` above to confirm what was requested by user?